src/server/db/mongo/user-privat-provider.js

```diff
@@ -0,0 +1,157 @@
+import UserPrivateModel from './models/user-private.js'
+import logger from '../../../imports/pino-logger'
+import { type UserRecord } from '../../../imports/types'
+
+class UserPrivate {
+  constructor() {
+    this.model = UserPrivateModel
+  }
+
+  /**
+   * Get new nonce after increment
+   *
+   * @param {UserRecord} user
+   *
+   * @returns {Promise<*>}
+   */
+  async isDupUserData(user: UserRecord): boolean {
+    const { email, mobile } = user
+    let result = null
+
+    if (email) {
+      result = await this.model.findOne({ email })
```

Comment:
use countDocuments/find with (query,{limit:1}) if you use find only return the _id field

Reply:
done